cmd/gossamer/genesis.go

```diff
@@ -31,7 +32,7 @@ func loadGenesis(ctx *cli.Context) error {
 		return err
 	}
 
-	log.Info("🕸\t Initializing node", "genesisfile", fp, "datadir", fig.Global.DataDir, "name", gen.Name, "id", gen.Id, "protocolID", gen.ProtocolId, "bootnodes", gen.Bootnodes)
+	log.Info("🕸\t Initializing node", "genesisfile", fp, "datadir", fig.Global.DataDir, "name", gen.Name, "id", gen.Id, "protocolID", gen.ProtocolId, "bootnodes", common.BytesToStringArray(gen.Bootnodes))
```

Comment:
This log is overloaded. Maybe it would make sense to but some of this into a `Debug` log and separate the system config params from the actual genesis values

Reply:
okay, I moved the system config parameters into a debug log